app/assets/stylesheets/application.scss

```diff
@@ -51,5 +48,44 @@ body {
 	bottom: 0;
 	width: 100%;
 	height: 60px;
-	background-color: #f5f5f5;
+	background-color: rgba(46, 48, 58, 0.96);
```

Comment:
Color literals like `rgba(46, 48, 58, 0.96)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs